DOC-12351 Add or Remove Services and Rebalance (#3846)
couchbase/docs-server: modules/rest-api/pages/rest-set-up-services.adoc

> = Assigning Services to a New Node

Reviewer: `= Assigning Services to a New Single Node`

== See Also

> * The sequence of tasks divided into _initialization_ and _provisioning_ is explained in xref:rest-api:rest-cluster-init-and-provisioning.adoc[Cluster Initialization and Provisioning].

Reviewer 2: ```suggestion
* For more information about cluster initialization and provisioning with the REST API, see xref:rest-api:rest-cluster-init-and-provisioning.adoc[Cluster Initialization and Provisioning].
```

* For each of the other specific steps required in the provisioning process, see xref:rest-api:rest-configure-memory.adoc[Configuring Memory], xref:rest-api:rest-name-node.adoc[Naming a Node], xref:rest-name-cluster.adoc[Naming a Cluster], and xref:rest-api:rest-establish-credentials.adoc[Establishing Credentials].
Specifically, see xref:rest-api:rest-configure-memory.adoc[Configuring Memory] for information on allocating memory per service, and on the default minimum values per service.

* For further information on initialization and provisioning -- using the UI, the CLI, and the REST API -- see xref:manage:manage-nodes/node-management-overview.adoc[Manage Nodes and Clusters].

> * For information about assigning non-Data Services to an existing node using the REST API, see xref:rest-api:rest-set-up-services-existing-nodes.adoc[Assigning Services to an Existing Node].

Reviewer 2: ```suggestion
* For more information on initialization and provisioning -- using the UI, the CLI, and the REST API -- see xref:manage:manage-nodes/node-management-overview.adoc[Manage Nodes and Clusters].

* For more information about assigning non-Data Services to an existing node using the REST API, see xref:rest-api:rest-set-up-services-existing-nodes.adoc[Assigning Services to an Existing Node].
```
